.vib/harbor-notary-signer/goss/vars.yaml

```diff
@@ -0,0 +1,3 @@
+binaries:
+  - notary-signer
```

Comment:
As done for `harbor-notary-server`, we should also check the `migrate` binary exists.